### PR TITLE
DEVX-2100: REST Proxy code example

### PR DESCRIPTION
**Pull request by ybyzek**
This jira adds REST proxy code example to https://docs.confluent.io/current/tutorials/examples/clients/docs/clients-all-examples.html

Validated with CCloud (and CCSR), though it's intended to run on any CP install as well

Docs staging available at https://github.com/confluentinc/docs/pull/5714

### CODE REVIEW

**Review by JimGalasyn — APPROVED** [review]
LGTM!

File: clients/cloud/rest-proxy/consume-ccsr.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+     http://localhost:8082/consumers/cg2 | jq .
+
+# Subscribe to a topic
+docker-compose exec rest-proxy curl -X POST \
+     -H "Content-Type: application/vnd.kafka.v2+json" \
+     --data '{"topics":["test2"]}' \
+     http://localhost:8082/consumers/cg2/instances/ci2/subscription | jq .
+
+# Consume some data using the base URL in the first response.
+# Note: Issue this command twice due to https://github.com/confluentinc/kafka-rest/issues/432
```
Comment (ghost) [inline clients/cloud/rest-proxy/consume-ccsr.sh:16]: I assume this is still an problem? It wasn't clear reading through https://github.com/confluentinc/kafka-rest/issues/432 whether the issue still exists or not.

**Review by ghost — APPROVED** [review]
LGTM

File: clients/cloud/rest-proxy/consume-ccsr.sh (hunk 1 above)
Comment (ybyzek) [reply clients/cloud/rest-proxy/consume-ccsr.sh:16]: @mikebin in my environment, https://github.com/confluentinc/kafka-rest/issues/432 is still an issue

**Review by rigelbm — APPROVED** [review]
LGTM!

Wow! Just awesome!! Thanks!!

**Pull request merged**